### DISCUSSION TITLE
Should ARCC be implemented

### DISCUSSION THREAD

**Discussion by srliao** [genshinsim/gcsim#778] [General]
This is a discussion thread for philosophical discussion on whether or not ARCC should be allowed in sim (due to potential "abusive" nature of requiring a macro for most people)

**Comment by skippi** [comment]
I still think ARCC is a DB/leaderboard issue, adding support for it isn't a problem. There are users who want to TC builds with ARCC or even RR cancels. For what gcsim strives to be, I don't see why we should discourage/prevent them from doing that.
